pyani/scripts/subcommands/subcmd_report.py

```diff
@@ -255,10 +255,10 @@ def subcmd_report(args: Namespace) -> int:
                 "Query description",
                 "Subject ID",
                 "Subject description",
-                "% identity",
+                "% identity (weighted)",
```

Comment:
We should remove the indication of `(weighted)` as this is not true for all comparison methods. I've done this in a new commit.